app/components/SourceSelector.vue.ts

```diff
@@ -12,6 +12,42 @@ import SlVueTree, {
 } from 'sl-vue-tree';
 import { $t } from 'services/i18n';
 
+const widgetIconMap = {
+  'Alert Box': 'fas fa-bell',
+  'Stream Boss': 'fas fa-gavel',
+  'Event List': 'fas fa-th-list',
+  'The Jar': 'fas fa-beer',
+  'Stream Labels': 'fas fa-file-alt',
+  'Donation Ticker': 'fas fa-ellipsis',
+  'Chat Box': 'fas-fa-comments',
+  'Viewer Count': 'fas fa-eye',
```

Comment:
we use the same icon for the visible/invisible scene items, how it's looking now?

Reply:
it's actually a slightly different icon, let me post a screenshot

Reply:
Oh, I see. We used the same icon before redesign but now it's different